hooks/dynamo

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```

Comment:
Do we need a separate script? 🤔

Reply:
Ah doesn't like we do, will remove and place the command directly in `CMD`

Reply:
Actually oops forget what I just said, we do need it. I ran into issues with getting the port env variable to work without having it setup like this